src/grammar.ts

```diff
@@ -107,10 +109,11 @@ export class TermSet {
     this.terms = this.terms.filter(t => t.terminal || t.preserve || rules.some(r => r.name == t || r.parts.includes(t)))
 
     let names: {[id: number]: string} = {}
-    let nodeTypes = [this.error, this.top]
+    let nodeTypes = [this.error, ...this.tops]
 
     this.error.id = T.Err
-    this.top.id = T.Top
+    for (const term of this.tops) term.id = T.Top
```

Comment:
I don't think this'll work—each term needs its own id, so we'll have to stop using a hard-coded top id.

Reply:
Fixed via https://github.com/lezer-parser/lezer-generator/pull/1/commits/1298f5d6659b87665bad111b25f212d7a0938316.